src/visualization/src/handler.cpp

```diff
@@ -0,0 +1,104 @@
+#include "handler.h"
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "include/base/cef_bind.h"
+#include "include/cef_app.h"
+#include "include/views/cef_browser_view.h"
+#include "include/views/cef_window.h"
+#include "include/wrapper/cef_closure_task.h"
+#include "include/wrapper/cef_helpers.h"
+
+namespace {
+  Handler* g_instance = NULL;
+}
+
+Handler::Handler(bool use_views) : use_views_(use_views), is_closing_(false) {
+  DCHECK(!g_instance);
+  g_instance = this;
+}
+
+Handler::~Handler() {
+  g_instance = NULL;
+}
+
+Handler* Handler::GetInstance() {
+  return g_instance;
+}
+
+void Handler::OnTitleChange(CefRefPtr<CefBrowser> browser, const CefString& title) {
+  CEF_REQUIRE_UI_THREAD();
+  if (use_views_){
```

Comment:
It's false in out application since we use X11 specifically. If someone were to use this Handler in a different context, maybe running the viz in a headless state (Not entirely sure if that's an example) then they could still bind a browser instance to the CEF window.

Reply:
I see. Makes sense to leave it in (we may want to support headless for either testing or programmatic use purposes), but please add a comment next to the variable declaration that explains what it's for.